### ISSUE TITLE
The fromNumber doesn't work in 0.9.0

### ISSUE THREAD

**Issue by neufreescale** [ringcentral/ringcentral-web-phone#369]
The "fromNumbe"r works well before 0,9.0.
It always uses the company default number in 0.9.0, although I set other numbers.

```
var session = webPhone.userAgent.invite('PHONE_NUMBER', {
    fromNumber: 'PHONE_NUMBER', // Optional, Company Number will be used as default
    homeCountryId: '1' // Optional, the value of
});
```

**Comment by tylerlong** [comment]
@embbnux  Did you reproduce the issue?

**Comment by embbnux** [comment]
Yes, it is reproduced. We should fix this

**Comment by tylerlong** [comment]
Released [1.0.0-beta2](https://www.npmjs.com/package/ringcentral-web-phone/v/1.0.0-beta2)

**Comment by tylerlong** [comment]
For those who don't want to upgrade to 1.0.0 beta versions, version 0.9.1 also has this issue fixed.
